### PR TITLE
changes made base on feedback

### PR DESCRIPTION
**Pull request by shabtaisharon**

### CODE REVIEW

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/PerformanceCommand.java [hunk 1]
```diff
@@ -184,13 +196,17 @@ public void objectCompleted(final String s) {
...
-                System.out.print(String.format("\rGet Statistics: (%d/%d MB), files (%d/%d), Time (%.03f sec), Mbps (%.03f), Highest Mbps (%.03f sec)",
-                        content, numberOfMB, numberOfFiles, totalNumberOfFiles, time, mbps, highestMbps));
+                messagePrefix = "Putting";
+            }
+            else {
+                messagePrefix = "Getting";
             }
+
+
+            System.out.print(String.format("\r%s Statistics: (%d/%d MB), files (%d/%d), Time (%.03f sec), MBps (%.03f), Highest MBps (%.03f sec)",
```
Comment (rpmoore) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/PerformanceCommand.java]: The Highest MBps does not need to have `sec` after the value is printed.

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/PerformanceCommand.java [hunk 2]
```diff
@@ -71,7 +74,16 @@ public PerformanceResult call() throws Exception {
...
 
         try {
-            helpers.ensureBucketExists(this.bucketName);
+            try {
+                final PutBucketRequest request = new PutBucketRequest(bucketName);
+                getClient().putBucket(request);
+            } catch(final FailedRequestException e) {
+                this.doNotDelete = true;
+                if (e.getStatusCode() == 409) {
+                    throw new CommandException("Bucket " + bucketName + " already exists. To avoid any conflict please use a non-existent bucket.");
```
Comment (rpmoore) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/PerformanceCommand.java]: `conflict` should be `conflicts`

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/PerformanceCommand.java (hunk 1 above)
Comment (shabtaisharon) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/PerformanceCommand.java]: done

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/PerformanceCommand.java (hunk 2 above)
Comment (shabtaisharon) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/PerformanceCommand.java]: done

**Pull request merged**